src/history.js

```diff
@@ -278,15 +278,16 @@ class History extends Emitter {
     console.assert(action, 'History should never reconcile ' + typeof action)
 
     let focus = action
-
     while (focus) {
-      this._emit('update', focus)
+      if (focus.command !== START) {
```

Comment:
I think a good next step for History is to figure out how to remove START, BIRTH, and ADD_DOMAIN